src/main/java/com/example/demo/model/entities/UserEntity.java

```diff
@@ -0,0 +1,88 @@
+package com.example.demo.model.entities;
+
+import com.example.demo.model.Role;
+import jakarta.persistence.*;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.Date;
+import java.util.Objects;
+
+@Entity(name="users")
+//@Table(name="users")
+public class UserEntity {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id", nullable = false)
+    private long id;
+
+    private String name;
+
+    private int age;
+
+    private Date createdAt;
+
+    private Date updatedAt;
```

Comment:
1. Move createdAt and updatedAt to the last position of class fields.
2. Change the type to `LocalDateTime`.
3. Instead of manual creation of time, use `@CreationTimestamp` and `@UpdateTimestamp`.